app/lib/features/chat/pages/room_profile_page.dart

```diff
@@ -172,24 +170,15 @@ class _RoomProfilePageState extends ConsumerState<RoomProfilePage> {
         // Invite
         myMembership.when(
           data: (membership) {
+            if (membership == null) return const SizedBox();
```

Comment:
The reason why AsyncValue exists here, is to allow transitioning over various future value states. This is already being handled as previously in ```onError()``` callback below. No need to put error state here :)